Firmware/Marlin_main.cpp

```diff
@@ -8279,7 +8279,7 @@ void get_coordinates()
   for(int8_t i=0; i < NUM_AXIS; i++) {
     if(code_seen(axis_codes[i]))
     {
-      bool relative = axis_relative_modes[i] || relative_mode;
+      bool relative = axis_relative_modes[i];
```

Comment:
wasn't axis_relative_modes array used for remembering the previous state for quick return from relative mode?
It looks that way (judging from the fact, that there is a separate array of axis' relative flags + one global relative flag for all of them).
Or was it just a relict of the past?

Reply:
@leptun explained: Probably some old forgotten array/bool that was used incorrectly. We now do it the right way